script.js

```diff
@@ -69,7 +70,23 @@ function showTitle() {
   showTitleWrapper.appendChild(showTitle);
```

Comment:
Because you have only one element (showTitle -h1) inside this div (showTitleWrapper), I don't think it was necessary to create a parent div(line 62). You could easily manipulate the h1 without the div. It is not something wrong but it's more work you had to put in.